### ISSUE TITLE
[Bug] Category of bridged devices is always `Other`

### ISSUE THREAD

**Issue by burmistrzak** [itavero/homebridge-z2m#903]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I was inspecting individual devices exposed through `homebridge-z2m`, and noticed that all devices have the same HAP accessory category of `Other`.

This behavior can be attributed to the following line:
https://github.com/itavero/homebridge-z2m/blob/cb71fd79e0890c9f6e8bf59214d27d6036e949d6/src/platform.ts#L598

Because we're not explicitly specifying an accessory category, a default category is used by Homebridge. I'm not sure _how_ relevant this is for bridged devices, but it might help in cases like #869... 🤔

### This plugin

1.11.0-beta.6

### Homebridge

1.7.0

### Zigbee2MQTT

1.39.0-dev

**Comment by itavero** [comment]
Any feature request you are proposing?

**Comment by itavero** [comment]
> maybe we can use the Z2M converter type as a hint for the correct accessory category?

Yes, but a single accessory can use multiple converters, so they should somehow have a priority between them.
